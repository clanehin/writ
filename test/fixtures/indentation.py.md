Leading indentation on reference lines is preserved.

```py
def print_lines(lines):
    #:: for statement ::#
        #:: print statement ::#
```

```py
#== for statement ==#
for line in lines:
```

```py
#== print statement ==#
print line
```
