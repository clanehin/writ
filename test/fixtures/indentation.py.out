def print_lines(lines):
    for line in lines:
        print line
